tests/foreman/ui_airgun/test_dashboard.py

```diff
@@ -0,0 +1,370 @@
+    content_view = entities.ContentView(organization=org).create()
+    content_view.publish()
+    promote(content_view.read().version[0], lc_env.id)
+    with session:
+        session.organization.select(org_name=org.name)
+        values = session.dashboard.search('lifecycle_environment={}'.format(
+            lc_env.name))
+        assert content_view.name in values['ContentViews'][
+            'content_views'][0]['Content View']
+        entities.LifecycleEnvironment(id=lc_env.id).delete()
+        values = session.dashboard.search('lifecycle_environment={}'.format(
+            lc_env.name))
+        assert values['HostConfigurationStatus']['total_count'] == 0
+        assert content_view.name in values['ContentViews'][
+            'content_views'][0]['Content View']
+
+
+@tier2
+def test_positive_host_collections(session):
+    """Check if the Host Collections widget displays list of host
+    collection in UI
+
+    :id: 1feae601-987d-4553-8644-4ceef5059e64
+
+    :Steps:
+
+        1. Make sure to have some hosts and host collections
+        2. Navigate Monitor -> Dashboard
+        3. Review the Host Collections Widget
+
+    :expectedresults: The list of host collections along with content host
+        is displayed in the widget
+
+    :CaseLevel: Integration
+    """
+    org = entities.Organization().create()
+    host = entities.Host(organization=org).create()
+    host_collection = entities.HostCollection(
+        host=[host],
+        organization=org,
+    ).create()
+    with session:
+        session.organization.select(org_name=org.name)
+        values = session.dashboard.read('HostCollections')
+        assert values['collections'][0]['Name'] == host_collection.name
+        assert values['collections'][0]['Content Hosts'] == '1'
+
+
+@tier3
+@run_in_one_thread
+@skip_if_not_set('clients', 'fake_manifest')
+@upgrade
+def test_positive_content_host_subscription_status(session):
+    """Check if the Content Host Subscription Status is working in the
+    Dashboard UI
+
+    :id: ce0d7b0c-ae6a-4361-8173-e50f6381194a
+
+    :Steps:
+
+        1. Register Content Host and subscribe it
+        2. Navigate Monitor -> Dashboard
+        3. Review the Content Host Subscription Status
+        4. Click each link:
+
+            a. Invalid Subscriptions
+            b. Valid Subscriptions
+
+    :expectedresults: The widget is updated with all details for Valid,
+        and Invalid Subscriptions
+
+    :CaseLevel: System
+    """
+    org = entities.Organization().create()
+    env = entities.LifecycleEnvironment(organization=org).create()
+    content_view = entities.ContentView(organization=org).create()
+    activation_key = entities.ActivationKey(
+        environment=env,
+        organization=org,
+    ).create()
+    # Using cdn repo as we need a rh repo (no matter are we in cdn or
+    # downstream) for subscription status to be ok
+    setup_org_for_a_rh_repo({
```

Comment:
Please refactor this test using [robottelo's products](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/products.py)
Example: https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui_airgun/test_errata.py#L68